### ISSUE TITLE
API: Can't query specific group and event endpoints

### ISSUE THREAD

**Issue by Kobo-one** [TheRestartProject/restarters.net#524]
I am trying to get specific group data of one of the groups in my network using the following endpoints:
`https://restarters.net/api/groups/59` and `https://restarters.net/api/events/708`. 

This group id and event id are available in the `https://restarters.net/api/groups/network` response for my token. 
But both detail endpoints (group and event) return a 404 - not found.

I am using a network provided to Statik.be, currently it only includes Maakbaar Leuven.

I assume this might have something to do with permissions?
Would it be possible to query the data of specific groups and events related to my network?

**Comment by Kobo-one** [comment]
@ngm Those seem to work, thanks!

It looks like the network endpoint is not yet available in the api/v2/. I'll probably need to use a mix of both v1 and v2?

**Comment by ngm** [comment]
> It looks like the network endpoint is not yet available in the api/v2/. I'll probably need to use a mix of both v1 and v2?

That's correct - it isn't in v2 yet.  Are you happy to use a mix of v1 and v2 for now?

(We could move it to v2, but we'd likely want to make some changes to it if we did.)

**Comment by Kobo-one** [comment]
Sorry for the late reply. The v2 network endpoints work great!

I had a question regarding the "image"-fields: 
In v1 every image was a full url to the public available image. In v2 however only the filename is returned.
To support v2 I was able to add "https://restarters.net/uploads" to the filename to fix this. But I wonder if it might be more useful if the api returned the full url instead?

**Comment by ngm** [comment]
@Kobo-one 

> But I wonder if it might be more useful if the api returned the full url instead?

Yes, I agree.  We'll change this.  When we do, it'll require a change to your client, however, to no longer add the 'https://restarters.net/uploads' prefix.  Is this OK for you?

cc @edwh

**Comment by Kobo-one** [comment]
@ngm , 

Yes no problem. When would this change be released? If you have no specific date a quick reminder here (after releasing) would be nice :)
